### ISSUE TITLE
Missing scrollbar in various views

### ISSUE THREAD

**Issue by bilelmoussaoui** [zed-industries/zed#15324]
### Summary

Show scroll bars in more places.

### Description

Scrollbar are missing from: 

- [x] Terminal panel (vertical)
- [x] Editor (horizontal)
- [X] Project Panel (horizontal)
- [X] Outline Panel  (vertical and horizontal)
- [x] Extensions page (vertical)
- [ ] Assistant Panel (vertical)
- [x] Markdown Preview
- [ ] Channels Panel (vertical)
- [ ] Find/Replace Search Box (vertical). See: https://github.com/zed-industries/zed/issues/15046

**Comment by notpeter** [comment]
This is true on all platforms (MacOS and Linux).  I also added Outline Panel and
Channels Panel to the list above.

**Comment by bilelmoussaoui** [comment]
It seems like even the "editor" panel lacks a horizontal scrollbar

**Comment by charlis-fl** [comment]
![image](https://github.com/user-attachments/assets/94fa1db0-7a06-4f81-b3a8-46d22a80e996)

There's no horizontal scrollbar as it should be, since there is more content on the right. Other than this Zed is perfect for me right now. And git changes ofcourse 🙂

**Comment by jonee** [comment]
There is no horizontal scroll as well for the Project Panel in case some file paths are long. 
<img width="332" alt="Screenshot 2024-08-20 at 12 08 58 AM" src="https://github.com/user-attachments/assets/a7c6bb6b-92d7-487d-b41c-27576c0618a5">

**Comment by RichardJECooke** [comment]
Also missing scrollbar in Markdown Preview

**Comment by notpeter** [comment]
> > Also missing scrollbar in Markdown Preview
> 
> 
> 
> Not sure if it should be a separate issue, but I cannot use PageUp/Down to scroll in the Markdown Preview either

File a separate issue, that should be an easy fix.
